docker-compose.prod.yml

```diff
@@ -13,7 +13,7 @@ services:
 
   web:
     container_name: web-prod
-    restart: always
+    restart: no
```

Comment:
I'm not sure if this is just my machine, but docker wouldn't let me start without putting the value in quotes (i.e. "no")